resources/datadog-integrations/software.rb.erb

```diff
@@ -8,6 +8,7 @@ if windows?
   require_relative "../../resources/datadog-integrations/validate_manifest"
 else
   require_relative "<%= project_dir %>/resources/datadog-integrations/validate_manifest"
+  whitelist_file /\/opt\/datadog-agent\/.+\.so.+/
```

Comment:
should we only whitelist files that are in `/opt/datadog-agent/embedded`, so that the health check still takes into account the files in `<integration_agent_folder>`?